src/app/(nousConnaitre)/dateCles/page.js

```
@@ -114,15 +114,15 @@ export default function Dates() {
             </div>
             <div className="relative grid place-items-start mt-4 -ml-[20rem] md:-ml-[20rem] sm:-ml-[15rem] xs:-ml-[11rem]">
               <ul
-                className="list-disc list-inside text-sm md:text-sm sm:text-xs xs:text-[9px]"
+                className="list-disc list-inside text-sm md:text-lg  sm:text-xs xs:text-[9px]"
                 style={{ lineHeight: "1" }}
               >
                 <li>
                   Ouverture de l'école Palmier <br /> (préscolaire et primaire)
```

ESLint

[error] `'` can be escaped with `&apos;`, `&lsquo;`, `&#39;`, `&rsquo;`.

(react/no-unescaped-entities)